elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpersUtil.java

```
@@ -0,0 +1,217 @@
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+/**
+ * Util class for Dynamic config helper module.
+ */
+public class DynamicConfigHelpersUtil {
+
+    public static final String SCHEMA_TYPE_TABLE = "table";
+    public static final String SCHEMA_TYPE_SECURITY = "security";
+    public static final String SCHEMA_TYPE_VARIABLE = "variable";
+    public static final String INVALID_ERROR_MSG = "Incompatible or invalid config";
+
+    private static final String TABLE_CONFIG_PATH = "tables/";
+    private static final String SECURITY_CONFIG_PATH = "security.hjson";
+    private static final String VARIABLE_CONFIG_PATH = "variables.hjson";
+
+    private static final String ELIDE_TABLE_VALIDATION_SCHEMA = "elideTableSchema.json";
+    private static final String ELIDE_SECURITY_SCHEMA = "elideSecuritySchema.json";
+    private static final String ELIDE_VARIABLE_SCHEMA = "elideVariableSchema.json";
+
+    private static final String CHAR_SET = "UTF-8";
+    private static final String NEW_LINE = "\n";
+
+    /**
+     * Converts hjson input string to valid json string.
+     * @param hjson : input config in hjson format
+     * @return valid json string
+     */
+    public String hjsonToJson(String hjson) {
+        return JsonValue.readHjson(hjson).toString();
+    }
+
+    /**
+     * Validates json config against json schema.
+     * @param schemaObj : schema object
+     * @param jsonConfig : config to be validated against schema
+     * @return true or false
+     */
+    public boolean isConfigValid(JSONObject schemaObj, String jsonConfig) {
+
+        try {
+            JSONObject data = new JSONObject(new JSONTokener(jsonConfig));
+            Schema schema = SchemaLoader.load(schemaObj);
+            schema.validate(data);
+            return true;
+        } catch (Exception e) {
```

Generally not a good practice to catch Exception (which can swallow runtime exceptions) except in uncaught handlers.  Let's be specific for what we expect to blow up.